(dp1
S'output'
p2
S'<class \'_mysql_exceptions.ProgrammingError\'> (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'}  ORDER BY sow.name\' at line 1")'
p3
sS'layer'
p4
S'C:\\Dev\\cssow_web2py\\src\\applications\\schemeofwork\\controllers/learningepisode.py'
p5
sS'code'
p6
S'<code object <module> at 07760380, file "C:\\Dev\\cssow_web2py\\src\\applications\\schemeofwork\\controllers/learningepisode.py", line 2>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "C:\\Dev\\cssow_web2py\\src\\gluon\\restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "C:\\Dev\\cssow_web2py\\src\\applications\\schemeofwork\\controllers/learningepisode.py", line 45, in <module>\n  File "C:\\Dev\\cssow_web2py\\src\\gluon\\globals.py", line 421, in <lambda>\n    self._caller = lambda f: f()\n  File "C:\\Dev\\cssow_web2py\\src\\applications\\schemeofwork\\controllers/learningepisode.py", line 15, in index\n    data = learningepisodeModel.get_all()\n  File "applications/schemeofwork/models/learningepisodeModel.py", line 116, in get_all\n    rows = db.executesql(select_sql)\n  File "C:\\Dev\\cssow_web2py\\src\\gluon\\packages\\dal\\pydal\\base.py", line 772, in executesql\n    adapter.execute(query)\n  File "C:\\Dev\\cssow_web2py\\src\\gluon\\packages\\dal\\pydal\\adapters\\__init__.py", line 67, in wrap\n    return f(*args, **kwargs)\n  File "C:\\Dev\\cssow_web2py\\src\\gluon\\packages\\dal\\pydal\\adapters\\base.py", line 413, in execute\n    rv = self.cursor.execute(command, *args[1:], **kwargs)\n  File "MySQLdb/cursors.py", line 205, in execute\n  File "MySQLdb/connections.py", line 36, in defaulterrorhandler\nProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'}  ORDER BY sow.name\' at line 1")\n'
p11
s.